src/components/va-alert/va-alert.css

```diff
@@ -26,16 +26,21 @@ div.alert {
   box-sizing: border-box;
 }
 
-div.alert:not(.full-width):not(.bg-only) {
+div:not(.full-width) div.alert:not(.bg-only) {
   border-left-style: solid;
   border-left-width: 10px;
 }
-div.alert.full-width.info,
-div.alert.full-width.warning {
+div.full-width,
+div.full-width {
   border-top-style: solid;
   border-top-width: 10px;
 }
 
+div.full-width div.alert {
+  max-width: 1000px;
+  margin: 0 auto;
```

Comment:
This is based on what I saw on va.gov:

![image](https://user-images.githubusercontent.com/2008881/135354781-923ee810-8897-48ab-9463-2ea5eb9b95b4.png)